aeron-client/src/main/cpp_wrapper/Context.h

```diff
@@ -568,6 +568,7 @@ class Context
         std::int32_t session_id,
         std::int64_t correlation_id)
     {
+        (void)async; // Not used in C++ callback
```

Comment:
Can have param defined as:         `aeron_async_add_subscription_t * /* async */,`
BTW `async` is not a great name here.

Reply:
That one should already be fixed.

With regards to naming, I agree.  I've pulled the name through from the existing C API function pointer definition.  I'm not sure what the most appropriate name should be.  However it is consistent at the moment.  If we want change it, we should do it across the definition and usages.